wire-ios-sync-engine/Tests/Source/Integration/APNSTests+Swift.swift

```diff
@@ -50,7 +51,16 @@ class APNSTests_Swift: APNSTestsBase {
         XCTAssertEqual(self.application?.applicationIconBadgeNumber, 0)
 
         // WHEN
-        userSession?.receivedPushNotification(with: noticePayloadForLastEvent(), completion: {})
+        guard let userSession = userSession else {
+            XCTFail("missing userSession")
+            return
+        }
+
+//        let exp = XCTestExpectation(description: "ds")
+        userSession.receivedPushNotification(with: noticePayloadForLastEvent(), completion: {
+//            exp.fulfill()
+        })
+//        wait(for: [exp], timeout: 5)
```

Comment:
To delete?

Reply:
yes I'll uncomment the test is disabled we still need to solve this one but it has is own ticket